android/sdl_android/src/androidTest/java/com/smartdevicelink/protocol/SdlPacketTests.java

```diff
@@ -60,6 +64,30 @@ public void testNull() {
         assertEquals(TestValues.GENERAL_STRING, sdlPacket.getTransportRecord().getAddress());
     }
 
+    @Test
+    public void testVehicleMake() {
+        HashMap<String, Object> testMap = new HashMap<>();
+        testMap.put(VEHICLE_MAKE, "sdl");
```

Comment:
@kostyaBoss On line 70 I would use actual values for `VEHICLE_MAKE`.  You could use `Ford` instead of `sdl`.